spring-cloud-generator/generate-library-list.sh

```diff
@@ -91,3 +93,4 @@ echo "Total in-scope client libraries: $count"
 
 # clean up
 rm -rf google-cloud-java/
+touch run-sanity-check/generate-library-list-finished
```

Comment:
Same here. I think just check that `library_list.txt` was created and has some valid content is sufficient.